superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -454,6 +455,11 @@ function ChartList(props: ChartListProps) {
       unfilteredLabel: t('All'),
       selects: registry
         .keys()
+        .filter(
+          k =>
+            !registry.get(k)?.behaviors?.includes(Behavior.NATIVE_FILTER) ||
+            isFeatureEnabled(FeatureFlag.DASHBOARD_FILTERS_EXPERIMENTAL),
```

Comment:
On second thought I think I'll break the `nativeFilterGate` out into a util and use that here, too